### DISCUSSION TITLE
2-3 Gläser Größen im Maker Tab

### DISCUSSION THREAD

**Discussion by Marvin0795** [AndreWohnsland/CocktailBerry#144] [Ideas]
Im Maker Tab drei zuvor definierte Gläser größen bildlich darstellen, um eine einfachere Auswahl zu treffen.

**Comment by AndreWohnsland** [comment]
Ich finde die Idee gut, würde aber gerne auch die Möglichkeit wie es jetzt ist behalten / ermöglichen. Ich schau mal ob ich eine Option zum an und abschalten einer neuen gui der Buttons für Volumen / Alkohol einführen kann beziehungsweise das machbar ist.

**Reply by Marvin0795** [reply]
Klasse! Ich bin auch gerade dabei, die Addon Funktionalitäten auszuprobieren 👍
